opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -102,6 +103,34 @@ def get_context(self) -> 'SpanContext':
             A :class:`.SpanContext` with a copy of this span's immutable state.
         """
 
+    def set_attribute(self: 'Span',
+                      key: str,
+                      value: 'types.AttributeValue'
+                      ) -> None:
+        """Sets an Attribute.
+
+        Sets a single Attribute with the key and value passed as arguments.
+        """
+
+    def add_event(self: 'Span',
+                  name: str,
+                  attributes: 'types.Attributes',
+                  ) -> None:
+        """Adds an Event.
+
+        Adds a single Event with the name and, optionally, attributes passed
+        as arguments.
+        """
+
+    def add_link(self: 'Span',
+                 context: 'SpanContext',
+                 attributes: 'types.Attributes',
+                 ) -> None:
+        """Adds a Link to another Span.
```

Comment:
```suggestion
        """Adds a Link to another span.
```

It's a pedantic change to be sure but the link here is to another conceptual span, not a `Span` in memory.